src/SmartComponents/User/User.js

```diff
@@ -85,7 +84,7 @@ class User extends Component {
         <DataListCheck aria-labelledby={ `check-user-${item.id}` } name={ `check-user-${item.id}` }/>
         <DataListCell>
           <StackItem>
-            <span id={ item.id }>{ `${item.first_name} ${item.last_name}` } </span>
+            <span id={ item.id }>{ `${item.username}` } </span>
```

Comment:
No need for template strings

Reply:
I left the template strings because it will eventually go back to first_name/last_name - but I will remove it and added it back when needed.